.storybook/StoryThemeProvider.tsx

```diff
@@ -0,0 +1,9 @@
+import { ThemeProvider } from '@terra-ui-packages/components';
```

Comment:
Do we care if files have the extension `.tsx` vs. `.ts`? That is, are we going to try to be strict and only use the `.tsx` extension when needed, or standardize on `.tsx` so that one doesn't have to rename files as they evolve?

Reply:
TypeScript requires `.tsx` in order to use JSX.
https://www.typescriptlang.org/docs/handbook/jsx.html

Reply:
Right, but do we care if we use the `.tsx` extension on files that don't have JSX. For example, I did have JSX in the story files, but then I ended up removing it when I was able to delete the custom render functions (and I didn't rename the file). And this file doesn't actually have JSX. Do we want to be strict about only using `.tsx` when there **is** JSX in the file?

Reply:
Well, NOW this file has JSX in it. But the story files still don't, and the fact that files can easily move back and forth between containing JSX is part of my question… it seems like it would be a pain to try to be strict about changing the extension back to `.ts` when a file no longer contains JSX.

Reply:
My guess would be that it'll mostly be a one way trip from `.ts` to `.tsx` as we convert existing components from react-hyperscript-helpers to JSX.

I'm more interested in the separation of concerns vs strictly whether a file contains JSX or not. For cases like the story files, where they're related to visual components, I'm fine with `.tsx` even if they don't contain JSX.

Reply:
Well, I just changed them back. :)